content/_index.md

```diff
@@ -1,26 +1,18 @@
 +++
-title = "Terraphim - Privacy Preserving AI Assistant"
+title = "Terraphim"
```

Comment:
Keeping title is important - this is what's going to be shown to users in search engines